### DISCUSSION TITLE
No example for PHDI in this tutorial file.

### DISCUSSION THREAD

**Discussion by fengjin98** [jamesjiadazhan/dietaryindex#288] [Q&A] [answered]
Thanks for the authors' outstanding work. I have some questions. I found no example for PHDI in this tutorial file. I am confused how to use NHANES data to calculate PHDI with your package. Can you add this part into the tutorial file? Many thanks!

**Comment by jamesjiadazhan** [comment]
Sorry. Currently, PHDI is currently not supported for NHANES.

**Reply by muzi18768163385-create** [reply]
Can we use FPED data to calculate PHDI?I am confused how to use NHANES data to calculate PHDI with dietaryindex R.

**Reply by jamesjiadazhan** [reply, marked as answer]
Please check this code and data repository. PHDI can be calcualted in NHANES but it has to be done in a complex way since all food groups need to be in grams. https://zenodo.org/records/18042895

> Can we use FPED data to calculate PHDI?I am confused how to use NHANES data to calculate PHDI with dietaryindex R.

**Comment by muzi18768163385-create** [comment]
Can we convert the FPED data into grams and then perform the PHDI calculation?
